awx/main/utils/encryption.py

```diff
@@ -125,3 +127,13 @@ def decrypt_field(instance, field_name, subfield=None):
             exc_info=True
         )
         raise
+
+
+def encrypt_dict(data, fields):
+    '''
+    Encrypts all of the dictionary values in `data` under the keys in `fields`
+    in-place operation on `data`
+    '''
+    encrypt_fields = set(data.keys()) & set(fields)
```

Comment:
Good choice w/ the `set`

For the less mathematically minded, could we use:

`encrypt_fields = set(data.keys()).interesection(fields)`

I can never remember set notation and have to go looking to make sure I've got it right :)